Library/Tests/Koala/KoalaTests.swift

```diff
@@ -66,7 +66,7 @@ final class KoalaTests: XCTestCase {
 
   func testDeviceFormatAndClientPlatform_ForIPhoneIdiom() {
     let client = MockTrackingClient()
-    let koala = Koala(client: client, loggedInUser: nil, device: MockDevice(userInterfaceIdiom: .Phone))
+    let koala = Koala(client: client, device: MockDevice(userInterfaceIdiom: .phone), loggedInUser: nil)
```

Comment:
this change is just due to the fact that swift 3 requires named args to be in the same order as the function's signature. `loggedInUser` has been swapped with `device`.